package.json

```diff
@@ -1,35 +1,39 @@
 {
   "name": "aggregated-public-information",
-  "private": true,
   "version": "0.1.0",
+  "private": true,
   "description": "A Strapi application",
+  "license": "MIT",
+  "author": {
+    "name": "A Strapi developer"
+  },
   "scripts": {
+    "build": "strapi build",
     "develop": "strapi develop",
     "start": "strapi start",
-    "build": "strapi build",
     "strapi": "strapi"
   },
-  "devDependencies": {},
   "dependencies": {
-    "@strapi/plugin-i18n": "4.14.0",
-    "@strapi/plugin-users-permissions": "4.14.0",
+    "@strapi/plugin-i18n": "4.20.4",
+    "@strapi/plugin-users-permissions": "4.20.4",
     "@strapi/provider-upload-aws-s3": "^4.14.5",
-    "@strapi/strapi": "4.14.0",
+    "@strapi/strapi": "4.20.4",
     "better-sqlite3": "^9.4.1",
     "pg": "^8.11.3",
     "pg-connection-string": "^2.6.2",
+    "react": "^18.0.0",
+    "react-dom": "^18.0.0",
+    "react-router-dom": "^5.2.0",
     "strapi-plugin-multi-select": "^1.2.2",
-    "strapi-plugin-slugify": "^2.3.5"
-  },
-  "author": {
-    "name": "A Strapi developer"
-  },
-  "strapi": {
-    "uuid": "679cf8f5-e045-4f0a-afc2-5bc6713ebdf0"
+    "strapi-plugin-slugify": "^2.3.5",
+    "styled-components": "^5.2.1"
   },
+  "devDependencies": {},
   "engines": {
-    "node": ">=18.0.0 <=18.x.x",
+    "node": ">=18.0.0 <=20.11.1",
     "npm": ">=6.0.0"
```

Comment:
**suggestion (non-blocking):** can we change this to yarn since we're not using npm?  you can use `yarn -v` to get the version number and then we should do something like `>=X.0.0` to indicate we can use that version or anything more up-to-date